openfecli/commands/plan_rbfe_network.py

```diff
@@ -78,13 +78,19 @@ def plan_rbfe_network_main(
 @COFACTORS.parameter(
     multiple=True, required=False, default=None, help=COFACTORS.kwargs["help"]
 )
+@YAML_OPTIONS.parameter(
+    multiple=False, required=False, default=None,
+    help=YAML_OPTIONS.kwargs["help"],
+)
```

Comment:
I'm (we're?) not familiar with the defaults, so being explicit here is useful to know what's happening

Reply:
If you want to actually be explicit, please list all standard defaults for [click.Option](https://click.palletsprojects.com/en/8.1.x/api/#click.Option) and [click.Parameter](https://click.palletsprojects.com/en/8.1.x/api/#click.Parameter), except for those overridden in `YAML_OPTIONS`. Personally, I think that's way too much information, so I don't think that being explicit is useful here.

In particular, please do not explicitly list anything that is just repeated from `YAML_OPTIONS`, such as `help=YAML_OPTIONS["help"]`. This is not informative to a reader. This got into our codebase because someone copied it from [this early code](https://github.com/OpenFreeEnergy/openfe/blob/2af8dcda2921ccfa131456bb291453ca3c487dbb/openfecli/commands/atommapping.py#L20-L21), and decided they didn't need the modification to the help. But as you can see from that code, the purpose of including it is adjust the default help for the parameter.